src/content-helper/editor-sidebar/class-editor-sidebar.php

```diff
@@ -136,7 +133,7 @@ public function init_features(): void {
 
 		// Initialize the features.
 		foreach ( $this->features as $feature ) {
-			$feature->run();
+				$feature->run();
```

Comment:
Extra tab that seems unneed?

```suggestion
			$feature->run();
```